internal/terraform/transform_targets.go

```diff
@@ -39,9 +41,26 @@ func (t *TargetsTransformer) Transform(g *Graph) error {
 		}
 	}
 
+	if len(t.ExcludeTargets) > 0 {
+
+		targetedNodes, err := t.selectExcludedNodes(g, t.ExcludeTargets)
+		if err != nil {
+			return err
+		}
+
+		for _, v := range g.Vertices() {
+			if targetedNodes.Include(v) {
+				log.Printf("[DEBUG] Removing %q, filtered by targeting.", dag.VertexName(v))
```

Comment:
It might be helpful to make this message different than the one above so they can be told apart in logs.
```suggestion
				log.Printf("[DEBUG] Removing %q, filtered by exclude targeting.", dag.VertexName(v))
```